.gitignore

```diff
@@ -52,4 +52,7 @@ scripts/netContain/bin
 # version
 version/version_gen.go
 
+# test files
+test/systemtests/cfg.json
```

Comment:
Not removed from git. I think it's better to have one sample cfg.json for reference